### ISSUE TITLE
BA in structure-less environments

### ISSUE THREAD

**Issue by HaisenbergPeng** [hku-mars/BALM#47]
Dear Author, thanks for your great work. 

As you might know, LiDAR BA may fall into degeneracy in structure-less environments. Do you recently have plans for solving such a problem by combining visual cues, sth like a combination of SFM and BALM?

**Comment by Zale-Liu** [comment]
Yes, you are right. Degeneration is challenge for single LiDAR. Adding visual information could help avoid degeneration and obtain colored point cloud, but I do not have any plans for vision yet.
